.github/workflows

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
```

Comment:
- [ ] You did well 😃at adding the linter files and folders, but I see that you forgot something to do😊 , I think you forgot to add inside the workflows the ``` linters.yml``` file, so the linter configuration will work the directories of linter configuration like that ``` .github/workflows/linters.yml```💯you can read about linter [Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)⤵️

Reply:
@osamaashraf6 hi, thanks for the feedback I really appreciate it, I have little doubt, I need only set de file linters.yml to the workflow folder. because when I click on the link I can see all configurations, or set the link and after that set all configurations to linters.yml?